entrypoint.sh

```diff
@@ -14,6 +14,7 @@ then
   ./manage.py init_db
   ./manage.py register_api
   cd rconweb 
+  ./manage.py makemigrations --no-input
```

Comment:
I might be wrong here, as I didn't work with that that often in the past: But isn't the database schema handled by alembic? We have an upgrade statement in L13 already?